keras_cv/layers/preprocessing/random_augmentation_pipeline.py

```diff
@@ -78,7 +78,13 @@ def __init__(
         self.augmentations_per_image = augmentations_per_image
         self.rate = rate
         self._layers_empty = False
-        if list(layers) == [] or layers is None:
+
+        if layers is None:
```

Comment:
I think we can just drop this check.  There’s no need for this, if users pass layers=none it will error out anyways in call.